docs/docker.md

```diff
@@ -0,0 +1,247 @@
+# Using the MMLSpark Docker Image
+
+## Quickstart: install and run the docker image
+
+Begin by installing [Docker for your OS][docker-products].  Then, to get the
+MMLSpark image and run it, open a terminal (powershell/cmd on Windows) and run
+
+    docker run -it -p 8888:8888 microsoft/mmlspark
+
+In your browser, go to <http://localhost:8888/> — you'll see the docker image
+EULA, and once you accept it the jupyter notebook interface will start.  To skip
+this step, add `-e ACCEPT_EULA=yes` to the docker command (before the
+`microsoft/mmlspark`).
```

Comment:
1. Maybe clarify that EULA is for Docker only and source is released under the MIT license? 
1. Explicitly add the full like for docker run with the -e ACCEPT_EULA flag instead of writing before the microsoft/mmlspark line?
1. Spell "docker" and "jupyter" as "Docker" and "Jupyter" respectively?